### PR TITLE
feat(examples): add hello_world, update `r/demo/event`

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

We don't have a clean & simple hello_world realm. I also updated the doc on the `r/demo/events` realm, and also renamed it to emit.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
</details>

### CODE REVIEW

File: examples/gno.land/r/demo/hello_world/hello.gno [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Package hello_world demonstrates the usage of the Render() function.
+// Render() can be called via the vm/qrender ABCI query off-chain to retrieve
+// realm state or any other custom data defined by the realm developer.
+// The vm/qrender query allows for additional data to be passed in with the call
+// which can be utilized as the path argument to the Render() function.
+// This allows developers to create different "renders" of their realms
+// depending on the data which is passed in, such as pagination, admin dashboards, and more.
+// Check out other example realms and packages for more advanced use-cases, such as
+// r/gnoland/blog, r/gnoland/events, r/demo/hof, etc.
```
Comment (moul) [inline examples/gno.land/r/demo/hello_world/hello.gno:9]: Sufficiently long to warrant additional time for enhancing comment wrapping.

Comment (moul) [inline examples/gno.land/r/demo/hello_world/hello.gno:9]:
```suggestion
```

I don't think these other examples are ideal for understanding Render args. 

We should create a single, comprehensive realistic realm that incorporates all the best practices and then link to it.

**Review by moul — APPROVED** [review]

File: examples/gno.land/r/demo/hello_world/hello.gno (hunk 1 above)
Comment (leohhhn) [reply examples/gno.land/r/demo/hello_world/hello.gno]: [78ad717](https://github.com/gnolang/gno/pull/3130/commits/78ad717f9d381e4183a24a732ce9e250d0461bec)

Comment (leohhhn) [reply examples/gno.land/r/demo/hello_world/hello.gno]: Ok, I'll write something up. I'll remove this sentence for now

**Pull request merged**